views/make/index.js

```diff
@@ -73,6 +73,13 @@ module.exports = view.extend({
         goPreviousView: function (e) {
             global.history.back();
         },
+        onBackClick: function () {
+            if (this.mode !== 'edit') {
+                this.goPreviousView();
+            } else {
+                this.page('/profile');
```

Comment:
I'm def open to ideas here, but this is basically the functionality from the previous slideout nav.